waspc/src/Wasp/Generator/DbGenerator/Common.hs

```diff
@@ -65,6 +91,25 @@ dbSchemaChecksumOnLastGenerateFileInDbRootDir = [relfile|schema.prisma.wasp-gene
 dbSchemaChecksumOnLastGenerateFileProjectRootDir :: Path' (Rel ProjectRootDir) (File DbSchemaChecksumOnLastGenerateFile)
 dbSchemaChecksumOnLastGenerateFileProjectRootDir = dbRootDirInProjectRootDir </> dbSchemaChecksumOnLastGenerateFileInDbRootDir
 
+prismaClientOutputDirEnvVar :: String
+prismaClientOutputDirEnvVar = "PRISMA_CLIENT_OUTPUT_DIR"
+
+databaseUrlEnvVar :: String
+databaseUrlEnvVar = "DATABASE_URL"
+
+prismaClientOutputDirInModuleDir :: Path' (Rel d) (Dir ServerRootDir)
+prismaClientOutputDirInModuleDir = [reldir|node_modules/.prisma/client|]
+
+serverPrismaClientOutputDirEnv :: (String, String)
+serverPrismaClientOutputDirEnv = modulePrismaClientOutputDirEnv serverRootDirFromDbRootDir
+
+webAppPrismaClientOutputDirEnv :: (String, String)
+webAppPrismaClientOutputDirEnv = modulePrismaClientOutputDirEnv webAppRootDirFromDbRootDir
+
+modulePrismaClientOutputDirEnv :: Path' (Rel DbRootDir) (Dir d) -> (String, String)
+modulePrismaClientOutputDirEnv moduleRootDirFromDbRootDir =
+  (prismaClientOutputDirEnvVar, SP.fromRelDir $ moduleRootDirFromDbRootDir </> prismaClientOutputDirInModuleDir)
```

Comment:
Some of these things are now being used outside of `DbGenerator`. I've thought about moving these constants somewhere else, but then noticed that the same applies to `ServerGenerator/Common` and `WebAppGenerator/Common` and decided to do it in a separate PR:
```
$ rg "Wasp.Generator.DbGenerator" -l | grep -v DbGenerator
Wasp/Generator.hs
Wasp/Generator/Setup.hs
Wasp/Generator/ServerGenerator/ConfigG.hs
Wasp/Generator/DockerGenerator.hs
```
```
$ rg "Wasp.Generator.ServerGenerator" -l | grep -v ServerGenerator
Wasp/Lib.hs
Wasp/Generator/Start.hs
Wasp/Generator/WebAppGenerator/ExternalAuthG.hs
Wasp/Generator/WebAppGenerator/OperationsGenerator.hs
Wasp/Generator/NpmInstall.hs
Wasp/Generator/DbGenerator/Common.hs
Wasp/Generator/DbGenerator/Jobs.hs
Wasp/Generator/DockerGenerator.hs
Wasp/Generator.hs
```
```
$ rg "Wasp.Generator.WebAppGenerator" -l | grep -v WebAppGenerator
Wasp/Lib.hs
Wasp/Generator/NpmInstall.hs
Wasp/Generator/ServerGenerator/ExternalAuthG.hs
Wasp/Generator/ConfigFile.hs
Wasp/Generator/Start.hs
Wasp/Generator.hs
```